src/test/java/org/folio/rest/impl/OaiPmhImplTest.java

```diff
@@ -267,13 +266,6 @@ void setUpBeforeEach() {
     RestAssured.config().decoderConfig(DecoderConfig.decoderConfig());
   }
 
-  @AfterEach
-  void cleanUp(VertxTestContext testContext) {
```

Comment:
Since all instances are linked via requestId which is unique in the scope of each single test case then the data conflict is impossible here, therefore the method should be deleted.